libraries/bxdf/translation/standard_surface_to_gltf_pbr.mtlx

```diff
@@ -17,6 +17,7 @@
     <input name="coat_roughness" type="float" value="0.1" />
     <input name="emission" type="float" value="0" />
     <input name="emission_color" type="color3" value="1, 1, 1" />
+    <input name="normal" type="vector3" value="0.5, 0.5, 1.0" />
```

Comment:
The value seems to come from the Standard Surface to UsdPreviewSurface translation graph, and I don't think it is correct in this context. We should use `defaultgeomprop="Nworld"` instead.